src/ixdat/exporters/csv_exporter.py

```diff
@@ -1,13 +1,20 @@
 """Classes for exporting measurement data"""
+from pathlib import Path
 
 
 class CSVExporter:
     """The default exporter, which writes delimited measurement data row-wise to file"""
 
-    def __init__(self, measurement=None, delim=",\t"):
+    def __init__(self, measurement=None, delim=",\t", default_v_list=None):
         """Initiate the exported with a measurement (Measurement) and delimiter (str)"""
         self.measurement = measurement
         self.delim = delim
```

Comment:
Feel like this could be "delimiter" without it getting in the way

Reply:
done.